sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -0,0 +1,29 @@
+"""Main file to store schema definition"""
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    Integer,
+)
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class UniversalSpinTable(Base):
```

Comment:
It looks like there's a mocking library for sqlalchemy. Is this something we might want to use for unit testing? https://pypi.org/project/alchemy-mock/

Reply:
That or may be this: https://pypi.org/project/pytest-sqlalchemy-mock/